### DISCUSSION TITLE
New to the game - PAPERMERGE__SECURITY__TOKEN_EXPIRE_MINUTES

### DISCUSSION THREAD

**Discussion by Maikpoe** [ciur/papermerge#621] [Q&A] [answered]
Hello everybody,

As I mentioned in the title, I'm very new to Linux (Ubuntu), Docker, and all the related tools I've learned about in the last few days to get Papermerge running.

I successfully created a cron job that takes documents scanned with our floor printer and imports them into Papermerge using papermerge-cli.

Now, I can't figure out where to place PAPERMERGE__SECURITY__TOKEN_EXPIRE_MINUTES to extend the token expiration beyond the default 6 hours.

Unfortunately, the documentation doesn't seem clear enough for me to understand.

Could anyone explain this to me?

Best regards,
Maik

**Comment by ciur** [comment, marked as answer]
Do you use docker compose for setup?
If yes, then you need to place it along other environment variables (e.g. among capitalized environment variables from screenshot below):

![Screenshot from 2024-07-16 21-06-44](https://github.com/user-attachments/assets/b59039a1-8016-4add-a69c-cbb55fb3973e)

[Link to docker compose setups](https://docs.papermerge.io/3.2/setup/docker-compose/#postgresql)

**Reply by Maikpoe** [reply]
Thank you for your rly fast reply sir and your time!  

Yes I'm using the docker composer setup.

So if understanding correctly I will add the below line in the environment section first ,  restarting the docker container and generating a new token afterwards? 

PAPERMERGE__SECURITY__TOKEN_EXPIRE_MINUTES=10080

What about the format of the line? Everything within the file is separated by : , in the documentation 
PAPERMERGE__SECURITY__TOKEN_EXPIRE_MINUTES is separated from his value with a = 
Maybe I'm missing some fundamentals to understand the difference or I'm just dump^^ 

BR 
Maik
